src/frontend/komponenter/Felleskomponenter/Skjemasteg/Skjemasteg.tsx

```diff
@@ -48,7 +67,7 @@ const Skjemasteg: React.FunctionComponent<IProps> = ({
     }, [forrigeÅpneSide]);
 
     return (
-        <div
+        <Container
             id={'skjemasteg'}
             className={classNames('skjemasteg', className)}
```

Comment:
Siden denne ble lagd om til en styled component, kanskje like gjerne flytte over stylingen som ligger for klassenavnet, så vi kan fjerne className her?

Reply:
Ja, glemte å fjerne skjemasteg klassenavnet. className trenger vi i tilfelle konsumenten styler denne.